src/current/_includes/v23.2/sidebar-data/self-hosted-deployments.json

```diff
@@ -603,6 +603,35 @@
                   }
                 ]
               },
+              {
```

Comment:
Where these docs are positioned right now, feels too prominent in the DR section. I think that these Cluster Virtualization docs, should move to one of three places in the nav:

1. Below the Physical Cluster Replication section.
2. Within the Physical Cluster Replication section (and at the bottom of that section).
3. Toward the bottom of the Self-Hosted Deployment section.

Not knowing what CV will be applicable to in the next release or two, I understand why it is under DR here. However, with upcoming DR tools, and docs work we want to do here, I don't think CV should be too prominent. So, I think we should tightly couple it to PCR, or not, and ensure the links are back and forth to each doc set (which you already have in place).

Reply:
Sounds good, I will slot the CV docs into the bottom of the PCR section for now.